src/main/java/net/snowflake/client/jdbc/SnowflakeType.java

```diff
@@ -424,9 +422,10 @@ public static SnowflakeType javaTypeToSFType(int javaType) throws SnowflakeSQLEx
         return ANY;
 
       default:
-        throw new SnowflakeSQLException(
+        throw new SnowflakeSQLLoggedException(
             SqlState.FEATURE_NOT_SUPPORTED,
             ErrorCode.DATA_TYPE_NOT_SUPPORTED.getMessageCode(),
+            /* session = */ null,
```

Comment:
you have `session` above but why setting to `null` ?

Reply:
need fix this

Reply:
sorry, good catch. Fixed.